modules/s3-bucket/versions.tf

```diff
@@ -6,9 +6,5 @@ terraform {
       source  = "hashicorp/aws"
       version = ">= 4.0"
     }
-    template = {
-      source  = "cloudposse/template"
-      version = ">= 2.2.0"
-    }
```

Comment:
@westonplatter Sorry I was wrong about removing this, it is being used elsewhere 😣

Reply:
all good. Seemed like a good idea to me as well. Reverted.